src/System.Numerics.Vectors/tests/Matrix4x4Tests.cs

```diff
@@ -0,0 +1,2549 @@
+            private Matrix4x4PlusFloat _b;
+        }
+
+        // A test to make sure the fields are laid out how we expect
+        [Fact]
+        public unsafe void Matrix4x4FieldOffsetTest()
+        {
+            Matrix4x4 mat = new Matrix4x4();
+
+            float* basePtr = &mat.M11; // Take address of first element
+            Matrix4x4* matPtr = &mat; // Take address of whole matrix
+
+            Assert.Equal(new IntPtr(basePtr), new IntPtr(matPtr));
+
+            Assert.Equal(new IntPtr(basePtr + 0), new IntPtr(&mat.M11));
+            Assert.Equal(new IntPtr(basePtr + 1), new IntPtr(&mat.M12));
+            Assert.Equal(new IntPtr(basePtr + 2), new IntPtr(&mat.M13));
+            Assert.Equal(new IntPtr(basePtr + 3), new IntPtr(&mat.M14));
+
+            Assert.Equal(new IntPtr(basePtr + 4), new IntPtr(&mat.M21));
+            Assert.Equal(new IntPtr(basePtr + 5), new IntPtr(&mat.M22));
+            Assert.Equal(new IntPtr(basePtr + 6), new IntPtr(&mat.M23));
+            Assert.Equal(new IntPtr(basePtr + 7), new IntPtr(&mat.M24));
+
+            Assert.Equal(new IntPtr(basePtr + 8), new IntPtr(&mat.M31));
+            Assert.Equal(new IntPtr(basePtr + 9), new IntPtr(&mat.M32));
+            Assert.Equal(new IntPtr(basePtr + 10), new IntPtr(&mat.M33));
+            Assert.Equal(new IntPtr(basePtr + 11), new IntPtr(&mat.M34));
+
+            Assert.Equal(new IntPtr(basePtr + 12), new IntPtr(&mat.M41));
+            Assert.Equal(new IntPtr(basePtr + 13), new IntPtr(&mat.M42));
+            Assert.Equal(new IntPtr(basePtr + 14), new IntPtr(&mat.M43));
+            Assert.Equal(new IntPtr(basePtr + 15), new IntPtr(&mat.M44));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void PerspectiveFarPlaneAtInfinityTest()
+        {
+            var nearPlaneDistance = 0.125f;
+            var m = Matrix4x4.CreatePerspective(1.0f, 1.0f, nearPlaneDistance, float.PositiveInfinity);
+            Assert.Equal(-1.0f, m.M33);
+            Assert.Equal(-nearPlaneDistance, m.M43);
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void PerspectiveFieldOfViewFarPlaneAtInfinityTest()
+        {
+            var nearPlaneDistance = 0.125f;
+            var m = Matrix4x4.CreatePerspectiveFieldOfView(MathHelper.ToRadians(60.0f), 1.5f, nearPlaneDistance, float.PositiveInfinity);
+            Assert.Equal(-1.0f, m.M33);
+            Assert.Equal(-nearPlaneDistance, m.M43);
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void PerspectiveOffCenterFarPlaneAtInfinityTest()
+        {
+            var nearPlaneDistance = 0.125f;
+            var m = Matrix4x4.CreatePerspectiveOffCenter(0.0f, 0.0f, 1.0f, 1.0f, nearPlaneDistance, float.PositiveInfinity);
+            Assert.Equal(-1.0f, m.M33);
+            Assert.Equal(-nearPlaneDistance, m.M43);
+        }
```

Comment:
The SkipOnTargetFramework logic is not working. These 3 tests are getting executed in .NET Framework and failing. Since the src csproj has a netcoreapp2.0 tfm just to cover the `_._` case, I don't think I need to add that tfm to the tests csproj. We only need to test `$(NetFrameworkMinimum)`, correct? So I can just delete these 3 tests:

```
[xUnit.net 00:00:01.68]     System.Numerics.Tests.Matrix4x4Tests.PerspectiveFieldOfViewFarPlaneAtInfinityTest [FAIL]
  Failed System.Numerics.Tests.Matrix4x4Tests.PerspectiveFieldOfViewFarPlaneAtInfinityTest [10 ms]
  Error Message:
   Assert.Equal() Failure
Expected: -1
Actual:   NaN
  Stack Trace:
     at System.Numerics.Tests.Matrix4x4Tests.PerspectiveFieldOfViewFarPlaneAtInfinityTest() in C:\Users\calope\source\repos\maintenance-packages\src\System.Numerics.Vectors\tests\Matrix4x4Tests.cs:line 2535
[xUnit.net 00:00:01.72]     System.Numerics.Tests.Matrix4x4Tests.PerspectiveOffCenterFarPlaneAtInfinityTest [FAIL]
  Failed System.Numerics.Tests.Matrix4x4Tests.PerspectiveOffCenterFarPlaneAtInfinityTest [1 ms]
  Error Message:
   Assert.Equal() Failure
Expected: -1
Actual:   NaN
  Stack Trace:
     at System.Numerics.Tests.Matrix4x4Tests.PerspectiveOffCenterFarPlaneAtInfinityTest() in C:\Users\calope\source\repos\maintenance-packages\src\System.Numerics.Vectors\tests\Matrix4x4Tests.cs:line 2545
[xUnit.net 00:00:01.73]     System.Numerics.Tests.Matrix4x4Tests.PerspectiveFarPlaneAtInfinityTest [FAIL]
  Failed System.Numerics.Tests.Matrix4x4Tests.PerspectiveFarPlaneAtInfinityTest [1 ms]
  Error Message:
   Assert.Equal() Failure
Expected: -1
Actual:   NaN
  Stack Trace:
     at System.Numerics.Tests.Matrix4x4Tests.PerspectiveFarPlaneAtInfinityTest() in C:\Users\calope\source\repos\maintenance-packages\src\System.Numerics.Vectors\tests\Matrix4x4Tests.cs:line 2525
```

Reply:
We should try to make SkipOnTargetFramework work, can you make sure to file an issue for that?

Reply:
Done https://github.com/dotnet/maintenance-packages/issues/64